Makefile

```diff
@@ -0,0 +1,24 @@
+build:
+	pipenv run ./scripts/build.sh
+
+lint:
+	pipenv run ./scripts/run_lint.sh
+
+test:
+	pipenv run ./scripts/run_tests.sh
+
+test-unit:
+	pipenv run ./scripts/run_tests_unit.sh
+
+test-functional:
+	pipenv run ./scripts/run_tests_functional.sh
+
+run:
+	ln -sf .development.env .env
+	pipenv run flask run
+
+dev-compose-up:
+	docker-compose -f docker-compose-dev.yml up -d
```

Comment:
You need to change references to `go-launch-a-survey` to `eq-questionnaire-launcher` in both compose files.
@bitdivision must have missed them when forking the launcher repo